pkg/imagebuilder/client.go

```diff
@@ -103,8 +103,8 @@ type S3UploadStatus struct {
 
 // OsTree struct to get the metadata response : Saurabh
 type OsTree struct {
-	OstreeCommit string `json:"ostree_commit"`
-	Packages     InstalledPackage
+	OstreeCommit     string             `json:"ostree_commit"`
+	InstalledPackage []InstalledPackage `json:"packages"`
```

Comment:
```suggestion
	InstalledPackages []InstalledPackage `json:"packages"`
```

Can we call this field InstalledPackages to make it clear that is it a list?

Reply:
sure, makes sense